main.py

```diff
@@ -196,6 +196,37 @@ def h_unfetch(args):
 		if not success:
 			print("Could not find package '{}'".format(name))
 
+def h_info(args):
+	bot = halibot.Halibot()
+	bot._load_config()
+
+	if args.object_name:
+		# Show configuration of specific object
+
+		conf = bot.config["agent-instances"].get(args.object_name)
+		conf = bot.config["module-instances"].get(args.object_name) if not conf else conf
+		if not conf:
+			print("No such agent or module")
+			return
+
+		print(f"\n{args.object_name}: ({conf['of']})")
+		for k in conf:
+			if k != "of":
+				print(f"  {k}: {conf[k]}")
+	else:
+		# Show all configured objects
+		print("\nConfigured agents:")
+		agents = bot.config.get("agent-instances")
```

Comment:
We should probably catch the case where there isn't an `"agent-instances"` key in the config for some reason. Probably will never happen though with config validation, but maybe this should just include a default in the `.get("agent-instances", [])`

Reply:
Yeah, your right. Probably shouldn't bother to show anything when there is no agents to show as well.